PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -432,6 +433,102 @@ local function CreateCamera()
 		return vrRotateSum
 	end
 
+	local cameraTranslationConstraints = Vector3.new(1,1,1)
```

Comment:
We might want to start splitting these scripts up based on input or something... starting to get long